.node-version

```diff
@@ -0,0 +1 @@
+20.5.0
```

Comment:
Since we're adding this file, we should put `node-version-file: .node-version` in the `actions/setup-node` step in .github/workflows/*.

Reply:
Great. Sounds good.

Reply:
This has been fixed.